_docs/install/openshift/dynamic-provisioning.md

```diff
@@ -0,0 +1,282 @@
+---
+layout: guide
+title: StorageOS Docs - OpenShift Dynamic Provisioning
+anchor: install
+module: install/openshift/dynamic-provisioning
+---
+
+# Dynamic Provisioning
+
+StorageOS volumes can be created on-demand through dynamic provisioning.
+
+1. Create the secret needed to authenticate against the StorageOS API.
+1. Adminstrators create storage classes to define different types of storage.
```

Comment:
same as k8s comment, Helm will create the secret and storage class

Reply:
I've added a note to skip to step 3 if you used the helm chart.